### PR TITLE
Added a beginner's article on Blockchain

### PR DESCRIPTION
**Pull request by ridhiratan**
## Type of PR

- [x] Blog/Article
- [ ] Feature/Enhancement
- [ ] Bug Fix

## Description

Describe the changes made and why they were made instead of how they were made. List any dependencies that are required for this change.

## Related issues and discussion

#{Issue Number}

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] If you have multiple commits please combine them into one commit by squashing them.

- [ ] Read and understood the contribution guidelines at `devbae-blog/.github/CONTRIBUTING.md`.

### CODE REVIEW

File: _data/authors.yml [hunk 1]
```diff
@@ -123,3 +123,25 @@ pranshukharkwal:
...
+      url: "https://ridhiratan.github.io/ridhi/"
+    - label: "GitHub"
+      icon: "fab fa-fw fa-github"
+      url: "https://github.com/ridhiratan"
+    - label: "Linkedin"
+      icon: "fab fa-linkedin"
+      url: "https://www.linkedin.com/in/ridhi-ratan-a00795191/"
+    - label: "Instagram"
+      icon: "fab fa-fw fa-instagram"
+      url: "https://www.instagram.com/ridhi_2719/"
```
Comment (aashish-ak) [inline _data/authors.yml:147]: Add end of file newline.

File: _posts/2020-08-30-blockchain.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+Also, there’s this concept of **“51% Attack”**, according to which if more than 50% of the users go corrupt, the system would fall down and collapse, but this is a far too negligible possibility that more than 50% of the users would turn malicious. Or a single miner would form more than half of the part of the system .
+
+So in all and all we can say that it has the potential to change our vision in the way we trust third parties in our day-to-day life. Since it is a new topic it needs much modifications, everyday new algorithms are coming in the market, with time this technology would need upgradation and will also generate multiple opportunities in future. It gives us a solution to solve trust issues by managing the system all by ourselves. And probably has *the potential to revolutionise things in the coming years.*
+
+
```
Comment (aashish-ak) [inline _posts/2020-08-30-blockchain.md:91]: Keep only one end of line. Remove the extra ones.

**Review by aashish-ak — CHANGES_REQUESTED** [review]

File: _data/authors.yml [hunk 3]
```diff
@@ -124,6 +124,27 @@ pranshukharkwal:
       icon: "fab fa-instagram"
       url: "https://www.instagram.com/hellopranshu"
 
+
```
Comment (aashish-ak) [inline _data/authors.yml:127]: Remove this extra line.

File: _data/authors.yml [hunk 4]
```diff
@@ -124,6 +124,27 @@ pranshukharkwal:
...
+    - label: "GitHub"
+      icon: "fab fa-fw fa-github"
+      url: "https://github.com/ridhiratan"
+    - label: "Linkedin"
+      icon: "fab fa-linkedin"
+      url: "https://www.linkedin.com/in/ridhi-ratan-a00795191/"
+    - label: "Instagram"
+      icon: "fab fa-fw fa-instagram"
+      url: "https://www.instagram.com/ridhi_2719/"
 shreyasbapat:
```
Comment (aashish-ak) [inline _data/authors.yml:148]: Add a new line.

**Review by aashish-ak — CHANGES_REQUESTED** [review]

**Comment by aashish-ak** [comment]
@aroralakshya This blog is now looking fine, please give it a quick read and point out any discrepancies or errors you might find in the content, the design errors have been eradicated, now just the content needs to be reviewed once. Let me know if it's fine, will merger it. Also, good work @ridhiratan on this one, loved it, quick response to changes asked from you were cherry on the cake, will complete this review and will try to publish this blog by this weekend :)

**Review by aashish-ak — APPROVED** [review]

**Review by aroralakshya — COMMENTED** [review]
You have used some image from other sources. Can you write those sources below those images.

**Comment by ridhiratan** [comment]
> You have used some image from other sources. Can you write those sources below those images.

I picked them up from google, no specific site..... Should I directly mention the source as google?

**Comment by ridhiratan** [comment]
> You have used some image from other sources. Can you write those sources below those images.

Do check once, if what I have done is correct way of citing.

**Comment by aashish-ak** [comment]
Hi @ridhiratan , There's been a confusion between me and Lakshay regarding the citing of sources, as you're directly using the original links for the images, there's no need to cite the sources as the image itself is the source. So please remove the all the sources and we'll immediately publish the blog :)
Also very sorry for the delay in merging, we assure you this won't happen in future.

**Pull request merged**